packages/connect-thegraph/src/parsers/roles.ts

```diff
@@ -33,8 +28,9 @@ async function _parseRole(
     manager: role.manager,
     hash: role.roleHash,
     grantees,
-    artifact,
-    contentUri,
+    appId: app.appId,
```

Comment:
Includes the `appId` on the `Role` entity so we can override the artifact metadata for the apps without a repo on aragonPM.